tests/test_cleaning.py

```diff
@@ -0,0 +1,234 @@
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+            ],
+        ),
+        (
+            [
+                SimpleTransaction(
+                    transactionId="transaction_1",
+                    datetime=datetime(2022, 2, 1),
+                    amount=0.1,
+                    referenceText="reference_transaction_1",
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+                SimpleTransaction(
+                    transactionId="transaction_2",
+                    datetime=datetime(2022, 2, 1),
+                    amount=0.3,
+                    referenceText="internal_reference_1",
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+                SimpleTransaction(
+                    transactionId="transaction_3",
+                    datetime=datetime(2022, 2, 6),
+                    amount=-0.3,
+                    referenceText="internal_reference_1",
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+            ],
+            [
+                SimpleTransaction(
+                    transactionId="transaction_1",
+                    datetime=datetime(2022, 2, 1),
+                    amount=0.1,
+                    referenceText="reference_transaction_1",
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+                SimpleTransaction(
+                    transactionId="transaction_2",
+                    datetime=datetime(2022, 2, 1),
+                    amount=0.3,
+                    referenceText="internal_reference_1",
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+                SimpleTransaction(
+                    transactionId="transaction_3",
+                    datetime=datetime(2022, 2, 6),
+                    amount=-0.3,
+                    referenceText="internal_reference_1",
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+            ],
+        ),
+        (
+            [
+                SimpleTransaction(
+                    transactionId="transaction_1",
+                    datetime=datetime(2022, 2, 1),
+                    amount=0.1,
+                    referenceText="reference_transaction_1",
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+                SimpleTransaction(
+                    transactionId="transaction_2",
+                    datetime=datetime(2022, 2, 1),
+                    amount=0,
+                    referenceText="internal_reference_1",
+                    bankTransactionCode="dummy_transaction_code",
+                ),
+                SimpleTransaction(
+                    transactionId="transaction_3",
+                    datetime=datetime(2022, 2, 1),
+                    amount=-0.3,
```

Comment:
You could set this one to `0` as well to test the `transaction["amount"] != 0` logic in `cleaning.py`